whois-update/src/main/java/net/ripe/db/whois/update/handler/validator/organisation/LirRipeMaintainedAttributesValidator.java

```diff
@@ -34,8 +34,7 @@
             AttributeType.MNT_BY,
             AttributeType.ORG_NAME,
             AttributeType.ORG,
-            AttributeType.ORG_TYPE,
-            AttributeType.ABUSE_MAILBOX);
+            AttributeType.ORG_TYPE);
```

Comment:
We are allowing users to modify abuse-mailbox in the interim, is this acceptable?